testsuite/forge/src/test_utils/consensus_utils.rs

```diff
@@ -137,29 +137,46 @@ pub async fn test_consensus_fault_tolerance(
     let transactions: Vec<_> =
         join_all(validator_clients.iter().cloned().map(move |v| async move {
             let mut txns =
-                v.1.get_transactions(None, Some(1000))
-                    .await
-                    .map_err(|e| anyhow::anyhow!("{:?}", e))?
-                    .into_inner();
+                v.1.get_transactions(
+                    Some(if target_v < 1000 { 0 } else { target_v - 1000 }),
```

Comment:
nit: `cmp::max(target_v - 1000, 0)` would be concise and make it more obvious.

Reply:
yeah, but those are u64, so I would need to cast.

but actually I can just do target_v.saturating_sub(1000) which is even clearer.